build_modules/pubspec.yaml

```diff
@@ -26,10 +26,9 @@ dependencies:
 
 dev_dependencies:
   build_runner: ^1.0.0
-  build_test: ^0.10.0
-  build_vm_compilers: ">=0.1.0 <2.0.0"
+  #build_test: ^0.10.0
```

Comment:
note about commented-out things? Or just remove em?

Reply:
Decided to restore them and bump the min SDK instead.

I realized there isn't any value in supporting an older SDK than is supported by the only 2 packages that allow this version of `build_modules` 😝 

Technically `build_node_compilers` could end up supporting this version and older SDK than `build_vm_compilers`, but I think that's unlikely...